### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by kpmmmurphy**

### CODE REVIEW

File: alerter/src/main/java/com/tapadoo/alerter/Alert.java [hunk 1]
```diff
@@ -122,6 +123,8 @@ private void initView() {
         inflate(getContext(), R.layout.alerter_alert_view, this);
         setHapticFeedbackEnabled(true);
 
+        ViewCompat.setTranslationZ(this, Integer.MAX_VALUE);
```
Comment (otormaigh) [inline alerter/src/main/java/com/tapadoo/alerter/Alert.java]: Seems like overkill to get it to draw above the status bar, at least set it to half of max. Does this cause any issue with shadows or anything? 

Was it always being drawn below it or is this an Oreo issue?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Mon Mar 06 09:39:22 GMT 2017
+#Mon Dec 11 14:21:26 GMT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (otormaigh) [inline gradle/wrapper/gradle-wrapper.properties]: Don't forget to update the wrapper scripts as well.

File: alerter/src/main/java/com/tapadoo/alerter/Alert.java (hunk 1 above)
Comment (kpmmmurphy) [reply alerter/src/main/java/com/tapadoo/alerter/Alert.java]: Think of translation Z as a layering system, I want to ensure it's drawn over everything. So an arbitrarily large value is the best solution. No issues, as shadows come from the view's elevation value. 

The issue this solves is that when the Alerter is called early on in an Activity's lifecycle, it seems to get drawn under the status bar - In other cases, such as calling it from an on click listener, it functions as expected.

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (kpmmmurphy) [reply gradle/wrapper/gradle-wrapper.properties]: Should that have happened automatically?  Hmm..

Comment (otormaigh) [reply gradle/wrapper/gradle-wrapper.properties]: Not if you just manually changed the version number here. 

If you run the update from the command line
`./gradlew wrapper --gradle-version=4.1 --distribution-type all`

Or add this helper task to the root `build.gradle`
```
task wrapper(type: Wrapper) {
  gradleVersion = "4.1"
  distributionType = "all"
}
```
and run it with `./gradlew wrapper`, the wrapper jar and script should get updated when you do that.

Comment (kpmmmurphy) [reply gradle/wrapper/gradle-wrapper.properties]: Sweet, thanks. Did not know this was required.

**Pull request merged**